tools/pa11y.js

```diff
@@ -26,12 +26,12 @@ const options = {
 };
 
 /**
- * Returns an array of URLs with pa11y options.
+ * Returns an array of URLs with pa11y options. Join url via template literals.
  * @param {Array} testPaths - the paths we want to test.
  * @param {Array} options - the pa11y options we evaluate against.
  * @return {Array} pa11yPaths - a pa11y array to operate on.
  */
-const pa11yPaths = testPaths.map(path => pa11y(url.resolve(plPath, path), options));
+const pa11yPaths = testPaths.map(path => pa11y(`${plPath}${path}`, options));
```

Comment:
Let's change the `path` arg here to `testPath` (singular of the array it's mapping)